meeting-notes/2017-08-31--ipld-deep-dives.md

```diff
@@ -0,0 +1,78 @@
+# IPLD Deep Dives August 31
+
+**Moderator:** @diasdavid
+
+**Notetaker:** @jon-choi
+
+**Attendees** 
+* @stebalien
+* @Kubuxu
+* @Magik6k
+* @diasdavid
+* @jon-choi
+
+
+## Agenda
+
+<!-- Ensure notetaker is present before you begin -->
+- Start recording
+- Call for additional agenda items (moderator)
+- IPLD Interface Spec (@stebalien)
+- Sharded Maps on IPLD, HAMT, CHAMT (@Kubuxu)
+- IPLD Selectors/Transforms + UnixFS binary blobs (@stebalien)
+
+<!-- After each call, it is the responsibility of the notetaker to save the last
+version of the notes in a file in ipfs/pm/meeting-notes, by opening a branch and
+submitting a PR. -->
+
+## Notes
+
+* IPLD Interface Spec
+  * intro: go-ipld interface considerations @stebalien
+    * block: cid & raw data
+    * nodes are blocks (go-ipld)
+      * would like to add a method for unmarshal into a struct
+    * dag nodes
+      * don't have consistent format?
+    * this need stems from go being statically typed
```

Comment:
That is, `dagNode` in js-ipld don't have consistent types/interfaces. This doesn't work well (it can work, it just sucks) in statically typed languages so this interface isn't portable across languages.